### PR TITLE
CLOUD-526 document operator api via k8s rest api

### PR DESCRIPTION
**Pull request by fiowro**
[CLOUD-526](https://jira.percona.com/browse/CLOUD-526)

### CODE REVIEW

File: source/api.rst [hunk 1]
```diff
@@ -0,0 +1,557 @@
...
+
+.. code-block:: bash
+
+   Authorization: Bearer $KUBE_TOKEN
+
+**cURL Request:**
+
+.. code-block:: bash
+
+   curl -k -v -XPATCH 'https://$API_SERVER/apis/psmdb.percona.com/v1/namespaces/default/perconaservermongodbs/my-cluster-name' \
```
Comment (hors) [inline source/api.rst:308]:
```suggestion
   curl -k -v -XPATCH "https://$API_SERVER/apis/psmdb.percona.com/v1/namespaces/default/perconaservermongodbs/my-cluster-name" \
```

File: source/api.rst [hunk 2]
```diff
@@ -0,0 +1,557 @@
...
+.. code-block:: bash
+
+   Authorization: Bearer $KUBE_TOKEN
+
+
+**cURL Request:**
+
+.. code-block:: bash
+
+   curl -k -v -XPATCH 'https://$API_SERVER/apis/psmdb.percona.com/v1/namespaces/default/perconaservermongodbs/my-cluster-name' \
```
Comment (hors) [inline source/api.rst:378]:
```suggestion
   curl -k -v -XPATCH "https://$API_SERVER/apis/psmdb.percona.com/v1/namespaces/default/perconaservermongodbs/my-cluster-name" \
```

File: source/assets/code/api-update-cluster-image-request-json.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.. code-block:: json
+
+	{
+	"spec": { "image ": "percona/percona-server-mongodb-operator:1.5.0-mongod4.2" }
```
Comment (hors) [inline source/assets/code/api-update-cluster-image-request-json.txt:4]:
```suggestion
	"spec": { "image ": "percona/percona-server-mongodb:4.2.8-8" }
```

File: source/assets/code/api-update-cluster-image-response-json.txt [hunk 4]
```diff
@@ -0,0 +1,453 @@
...
+	      "allowUnsafeConfigurations":false,
+	      "backup":{
+		 "enabled":true,
+		 "image":"percona/percona-server-mongodb-operator:1.5.0-backup",
+		 "restartOnFailure":true,
+		 "serviceAccountName":"percona-server-mongodb-operator",
+		 "storages":null,
+		 "tasks":null
+	      },
+	      "image":"percona/percona-server-mongodb-operator:1.5.0-mongod4.2",
```
Comment (hors) [inline source/assets/code/api-update-cluster-image-response-json.txt:316]: why two times? is it correct?

File: source/assets/code/api-update-cluster-image-response-json.txt [hunk 5]
```diff
@@ -0,0 +1,453 @@
...
+	      "image ":"percona/percona-server-mongodb-operator:1.5.0-mongod4.2",
```
Comment (hors) [inline source/assets/code/api-update-cluster-image-response-json.txt:317]:
```suggestion
	      "image ":"percona/percona-server-mongodb:4.2.8-8",
```
and you need to change it in all files

File: source/assets/code/api-update-cluster-image-response-json.txt (hunk 4 above)
Comment (fiowro) [reply source/assets/code/api-update-cluster-image-response-json.txt:316]: it's the only one such place, sure it's a typo... fixed for now

**Review by delgod — APPROVED** [review]

**Pull request merged**